clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -2521,11 +2521,34 @@ public void testCommitOffsetMetadata() {
 
         AtomicBoolean success = new AtomicBoolean(false);
 
-        Map<TopicPartition, OffsetAndMetadata> offsets = singletonMap(t1p, new OffsetAndMetadata(100L, "hello"));
+        OffsetAndMetadata offsetAndMetadata = new OffsetAndMetadata(100L, "hello");
+        Map<TopicPartition, OffsetAndMetadata> offsets = singletonMap(t1p, offsetAndMetadata);
         coordinator.commitOffsetsAsync(offsets, callback(offsets, success));
         coordinator.invokeCompletedOffsetCommitCallbacks();
         assertTrue(success.get());
         assertEquals(coordinator.inFlightAsyncCommits.get(), 0);
+        Map<TopicPartition, OffsetAndMetadata> cache = coordinator.committedOffsetsCache();
+        assertEquals(cache.size(), 1);
+        assertEquals(cache.get(t1p), offsetAndMetadata);
```

Comment:
assertEquals method signature is `assertEquals(int expected, int actual)`. Putting the parameter in the correct order will output the reasonable error message if any.

Reply:
Also, could we assert cache is empty before we `commitOffsetsAsync`? i.e. 
```
assertTrue(cache.isEmpty());
coordinator.commitOffsetsAsync(...)
...
```

Reply:
Same comments applies to below tests

Reply:
Thanks, I like the suggested improvements. I'll adjust!